pamelalozano/src/main/webapp/maps.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Creates a map that shows landmarks around Google. */
+async function createMapChart(){
+
+  google.charts.load('current', {
+        'packages':['geochart'],
+        // Note: you will need to get a mapsApiKey for your project.
+        // See: https://developers.google.com/chart/interactive/docs/basic_load_libs#load-settings
+        'mapsApiKey': 'AIzaSyAcFI9_xPk0RWlT8w42YQfNyUhQbApYw3I'
+  });
+    
+      google.charts.setOnLoadCallback(drawRegionsMap);
+
+      async function drawRegionsMap() {
+        var data = google.visualization.arrayToDataTable(await fetchInfo('SL.IND.EMPL.FE.ZS'));
```

Comment:
What is `SL.IND.EMPL.FE.ZS`? Can you pull this out and make it a local variable with a descriptive name?

Reply:
Yes sorry, it's temporary though, I'm going to put a select input where you can choose what indicator the user wants to see, that is the temporary indicator code I was using.